css/src/editor.scss

```diff
@@ -167,15 +168,19 @@ div[class^="wp-block-block-lab-"] {
       font-size: 1em;
       color: rgba(0, 0, 0, 0.8);
     }
-  }
 
-  /* FetchInput component */
-  .bl-fetch-input {
-    max-width: $input-width;
```

Comment:
This style rule didn't allow `<FetchInput>` fields like Post and Taxonomy to respect the selected width, like 75% or 100%.

Reply:
Though maybe that's intended. We could just remove the 25%, 50%, 75%, 100% option from those fields.

Reply:
The 25% field width is still useful if you want to, say, have 4 text fields in a row. But having seen the fields taking up 100% of the space, I like it better that way.